ext/hal/st/stmemsc/a3g4250d_STdC/README.txt

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Mmmmh ok.
So I see two options now. Either keeping also the examples in the package or adapting all the
README (but there are many) stripping of all references to examples.

@erwango 
If you don't see issues I may go with option one which seems to be less impacting.

Reply:
I'm not sure adding examples juts because README files refer to it is a great option.
Do we really need these README files at all?
I don't see much added value in the information provided, this is only copy/paste from one to the other, and most of the information is not valid as referring to examples that you initially considered as superfluous (and I think you were right).
The only interesting information that differ from one file to the other is `H3LIS331DL DS rev3.0 `, which could be factorized in a upper file if needed.

Reply:
So, your preferred option would be:
1. Package w/o examples
2. Remove all README files under drivers
3. Keep the top README file w/o reference to example

Correct?
I'm ok with it. The only thing is that everytime I have to re-align the package there is an
extra effort (but not that much) to clean the package up.

Reply:
This is indeed indeed my preferred way to do it.
For the extra pain to do it, I suggest to spend few hours on a script to do it for  you.
I have this for cube packages update and this saves me painful hours each time.